packages/components/tabs/src/tabs.tsx

```diff
@@ -87,13 +87,17 @@ export default defineComponent({
 
     const nav$ = ref<TabNavInstance>()
     const panes = reactive<Record<number, TabsPaneContext>>({})
-    const currentName = ref<TabPanelName>(
-      props.modelValue ?? props.activeName ?? '0'
-    )
+    const currentName = computed<TabPanelName>({
+      get() {
+        return props.modelValue ?? props.activeName ?? '0'
+      },
+      set(value) {
+        emit(UPDATE_MODEL_EVENT, value)
+      },
```

Comment:
The `modelValue` and ~~`activeName`~~ attribute should be Control Props.

Should respond to value modifications outside the component, avoid modifications inside the component

Reply:
I emit update:model-value,it seems like respond to outside the component,how should I respond correctly

Reply:
The `set` attribute in the computed is redundant.